cupy/cuda/compiler.py

```diff
@@ -234,11 +234,7 @@ def _jitify_prep(source, options, cu_path):
     global _jitify_header_source_map_populated
     if not _jitify_header_source_map_populated:
         from cupy._core import core
-        _jitify_header_source_map = core._get_header_source_map()
-        _jitify_header_source_map_populated = True
-    else:
-        # this is already cached at the C++ level, so don't pass in anything
-        _jitify_header_source_map = None
+        jitify._add_sources(core._get_header_source_map())
```

Comment:
It seems `_jitify_header_source_map_populated` will never be set to `True` with the current change?

Reply:
Fixed! :+1: